src/proj.rs

```diff
@@ -96,6 +95,16 @@ pub enum ProjError {
     DownloadError(String, String, u8),
 }
 
+#[derive(Error, Debug)]
+pub enum ProjCreateError {
+    #[error("A nul byte was found in the PROJ string definition or CRS argument")]
+    ArgumentNulError,
```

Comment:
Nit: `NulError` gives you back the position of the NUL and the original string. I don't think it matters here, but it might be nice to follow the same design.

Reply:
Addressed in https://github.com/georust/proj/pull/98/commits/eec4a6b4ab8d43557fddfa626e35e1382e1d16cd